Egress Router CNI (OSDOCS-979)

mikemckiernan opened the pull request:
* The ER CNI plug-in is TP for the 4.7 release.

mikemckiernan commented:
Draft

/hold

mikemckiernan commented:
@jboxman , @danielmellado , @weliang1 , PTAL.

If this is technically accurate and as the definition of concision, then I'll take an approval or thumbs up now and add a URL to the content after https://github.com/openshift/openshift-docs/pull/28609 is merged.

danielmellado commented:
/lgtm

danielmellado commented:
Hi @mikemckiernan , left a few comments also on #28609, sounds good to me! I'll also reply to your JIRA and try to get a slot for a quick meeting on Fri! Thanks!

mikemckiernan commented:
@openshift/team-documention, PTAL.  After Daniel's review, I rebased and added an xref to the new topic.

/hold cancel

Applies to enterprise-4.7 and--small guess--milestone Future-Release.

Please add the label, milestone, and merge if it looks OK.

adellape commented on release_notes/ocp-4-7-release-notes.adoc:
```diff
@@ -166,6 +166,12 @@ With the addition of DNS support in the OVN-Kubernetes cluster network provider
+[id="ocp-4-7-ovn-kubernetes-egress-router-cni"]
+==== Egress router CNI (Technical Preview)
+
+The egress router CNI plug-in is introduced in Technical Preview. The plug-in
```
s/The plug-in enables you to/You can use the plug-in to/

(Per IBMSG)

adellape commented on release_notes/ocp-4-7-release-notes.adoc:
```diff
@@ -166,6 +166,12 @@ With the addition of DNS support in the OVN-Kubernetes cluster network provider
+==== Egress router CNI (Technical Preview)
+
+The egress router CNI plug-in is introduced in Technical Preview. The plug-in
+enables you to deploy an egress router in redirect mode. This egress router provides parity for OVN-Kubernetes compared to OpenShift SDN, but for redirect mode only. The plug-in does not perform in HTTP proxy or DNS proxy modes and this is a difference with the implementation for OpenShift SDN. For more information, see xref:../networking/ovn_kubernetes_network_provider/deploying-egress-router-ov ...
```
s/and this/, and this/

adellape reviewed (commented):
Looks good, couple tiny things. :+1:

mikemckiernan commented:
@adellape , thank you for the review and corrections.  It's ready for merge when you get a chance.
